amundsen-kube-helm/templates/helm/templates/amundsen-deployment.yaml

```diff
@@ -149,6 +149,10 @@ spec:
           - name: FRONTEND_BASE
             value: http://{{ .Values.frontEnd.FRONTEND_BASE }}
           {{ end }}
+          {{- if .Values.frontEnd.OIDC_OVERWRITE_REDIRECT_URI }}
+          - name: FLASK_OIDC_REDIRECT_OVERRIDE
```

Comment:
How are you configuring the FLASK_OIDC_REDIRECT_OVERRIDE ? 
Shouldn't it be FLASK_OIDC_OVERWRITE_REDIRECT_URI? as the official documentations says : OVERWRITE_REDIRECT_URI 

Here: https://flask-oidc.readthedocs.io/en/latest/

Reply:
It’s configured here: https://github.com/jornh/flaskoidc/commit/9368ec24ffacbd03590a03b2986fcc5792fa5deb You are right there’s a naming inconsistency across the layers. I choose to carry it forward to be compatible with the other fork of Verdan’s `flaskoidc` wrapper layer.